packages/passport-auth/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@keystone-alpha/passport-auth",
+  "description": "Provides Authentication Strategies based on PassportJS.",
+  "version": "1.0.0",
```

Comment:
The version should be set to `0.0.0` here, it will be bumped to `1.0.0` by the `changeset` tool when we next bump all our package versions.

Reply:
👍